infoschema/tables.go

```diff
@@ -99,9 +99,11 @@ const (
 	tableTiDBServersInfo                    = "TIDB_SERVERS_INFO"
 	tableClusterInfo                        = "CLUSTER_INFO"
 	// TableClusterConfig is the string constant of cluster configuration memory table
-	TableClusterConfig  = "CLUSTER_CONFIG"
-	tableClusterLoad    = "CLUSTER_LOAD"
-	tableTiFlashReplica = "TIFLASH_REPLICA"
+	TableClusterConfig     = "CLUSTER_CONFIG"
+	tableClusterLoad       = "CLUSTER_LOAD"
+	tableTiFlashReplica    = "TIFLASH_REPLICA"
+	tableClusterHardware   = "CLUSTER_HARDWARE"
+	tableClusterSystemInfo = "CLUSTER_SYSTEMINFO"
```

Comment:
@djshow832 @bb7133 @crazycs520 @wjhuang2016 How about `CLUSTER_SYSINFO`?

Reply:
I prefer `CLUSTER_SYSTEMINFO` to `CLUSTER_SYSINFO`, we don't need to abbreviate it here

Reply:
@wjhuang2016 OK